server/src/test/Diff.spec.ts

```diff
@@ -68,6 +68,10 @@ describe('Diff', () => {
     shouldReturnCorrespondingLineNumberForOldText('one_row', 0, 0);
   });
 
+  it('package_ref', () => {
+    shouldReturnCorrespondingLineNumberForNewText('package_ref', 1, 1);
```

Comment:
if you still decide to introduce a new function, then you need to write a lot of tests (like we have for old one) on it to be sure that the conversion works.

Reply:
I already check new function. I added new check to existing method shouldReturnCorrespondingLineNumber(...) in `server/src/test/Diff.spec.ts`. So I check position translation in both directions.

Reply:
Great, I didn't notice this, thanks!

Reply:
Check for both directions